packages/ra-ui-materialui/src/button/UpdateWithUndoButton.spec.tsx

```diff
@@ -95,11 +96,23 @@ describe('<UpdateWithUndoButton />', () => {
                     id: 123,
                     data: { views: 0 },
                     meta: undefined,
-                    previousData: { id: 123, title: 'lorem', views: 1000 },
+                    previousData: { id: 123, title: 'lorem', views: 500 },
                     resource: 'posts',
                 },
                 { snapshot: expect.any(Array) }
             );
         });
     });
+
+    it("should'nt open the show page caused by click propagation", async () => {
+        render(<WithAList />);
+        const startedUrl = global.window.location.pathname;
+        const resetButton = await screen.findByRole('button', {
+            name: 'Reset views',
+        });
+        screen.getByText('500');
```

Comment:
What this line is for?

Reply:
It is here to test that the click on the button "RESET VIEWS" works. If I have 500 views before click and 0 after the button works AND if the button works, the test to prevent click propagation has a real purpose.

Reply:
I was speaking about the previous line. But it's OK. Forget that :smile: My bad